active: true
control_frequency: 70
lookahead_distance: 0.45

# linear
xy_tolerance: 0.02
linear_max_vel: 1.0
linear_acceleration: 0.5
linear_kp: 0.9
linear_brake_vel: 0.15
linear_min_brake_distance: 0.2
linear_brake_distance_ratio: 0.2

# angular
theta_tolerance: 0.03
angular_max_vel: 1.2
angular_acceleration: 0.8
angular_kp: 0.8
angular_brake_distance: 0.15
angular_min_brake_distance: 0.15
angular_brake_distance_ratio: 0.15
